src/main/java/com/SAS/team/Team.java

```diff
@@ -134,18 +144,27 @@ public void setName(String name) {
      * The function return the home stadium of the team
      * @return
      */
-    public Stadium getHomeStadium() {
-        return homeStadium;
+    public Facility getHomeStadium() {
+        for(Facility facility: teamFacilities){
+            if (facility.facilityType == STADIUM){
```

Comment:
I think it should be facility.facilityType == facilityType .STADIUM.

Reply:
You're right. I'll fix it.